webpack.config.js

```diff
@@ -52,7 +76,7 @@ module.exports = {
         test: /\.worker\.js$/,
         use: {
           loader: "worker-loader",
-          options: { name: "[name].[ext]?[hash]" },
+          options: { inline: true, fallback: false },
```

Comment:
It was easiest to inline workers than to figure out how to make Webpack import stuff from the right paths. 🤷‍♂️